### PR TITLE
multiple improvements across the app

### PR DESCRIPTION
**Pull request by IlyaVi**

### CODE REVIEW

File: src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx [hunk 1]
```diff
@@ -6,42 +6,47 @@ import React, { useState, useRef, useEffect } from 'react';
...
   const classes = useStyles();
   const navigate = useNavigate();
   const captchaRef = useRef(null);
-  const faucetStore: FaucetStore = stores.Faucet;
   const walletStore: WalletStore = stores.Wallet;
   const [error, setError] = useState('');
   const [disabled, setDisabled] = useState(true);
   const [buttonText, setButtonText] = useState('Request');
   const [reachedMaxTransactionPerDay, setReachedMaxTransactionPerDay] = useState(false);
+  const [amountToTransfer, setAmountToTransfer] = useState(-1);
```
Comment (alexstolr) [inline src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx:31]: What is the meaning of -1?

File: src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx [hunk 2]
```diff
@@ -6,42 +6,47 @@ import React, { useState, useRef, useEffect } from 'react';
...
   const isDarkMode = useAppSelector(getIsDarkMode);
 
   useEffect(() => {
     setError('');
     setDisabled(true);
     setButtonText('Request');
+    const fetchAmountToTransfer = async () => {
+      const res = await getAmountToTransfer();
+      setAmountToTransfer(res || 10);
```
Comment (alexstolr) [inline src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx:40]: why 10?

**Review by alexstolr — APPROVED** [review]

File: src/app/common/stores/applications/Faucet/Wallet.store.ts [hunk 3]
```diff
@@ -45,94 +26,18 @@ class WalletStore extends BaseStore implements Wallet {
...
     if (wallet && connectedChain) {
       this.wallet = wallet;
-      const networkId = parseInt(String(connectedChain.id), 16);
       const address = wallet.accounts[0]?.address;
-      if (Number(networkId) !== NETWORKS.MAINNET) {
-        this.wrongNetwork = false;
-        await this.networkHandler(networkId);
+      // TODO: refuse connection if connecting with mainnet
+      if (!isMainnet()) {
+        // this.wrongNetwork = false;
```
[the remark this answers is not in the document]
Comment (IlyaVi) [reply src/app/common/stores/applications/Faucet/Wallet.store.ts:32]: we will need to fix Faucet and Distribution apps later. For now it is WIP

File: src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx (hunk 1 above)
Comment (IlyaVi) [reply src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx:31]: it should be undefined actually

**Review by axelrod-blox — APPROVED** [review]

File: src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx (hunk 2 above)
Comment (IlyaVi) [reply src/app/components/applications/Faucet/RequestForSsv/RequestForSsv.tsx:40]: old code leftover, removed

**Pull request merged**